integration-tests/cucumber-tests-platform-publiclighting/src/test/java/org/opensmartgridplatform/cucumber/platform/publiclighting/glue/steps/mocks/OslpDeviceSteps.java

```diff
@@ -1151,7 +1152,25 @@ public void theRegisterDeviceResponseContains(final Map<String, String> expected
         } else {
             assertThat(e.getMessage()).isEqualTo(getString(expectedResponse, PlatformPubliclightingKeys.MESSAGE));
         }
+    }
+
+
```

Comment:
The most likely reason is the difference between IntelliJ and Eclipse formatting / save actions. Since I made a change to the class and Eclipse performed formatting and save actions, the extra empty lines disappeared.